examples/Cargo.toml

```diff
@@ -42,6 +42,14 @@ path = "src/load_balance/client.rs"
 name = "load-balance-server"
 path = "src/load_balance/server.rs"
 
+[[bin]]
+name = "load-balance-client-discovery"
```

Comment:
What do you think about naming this `dynamic-load-balance`? Might be a bit more clear

Reply:
done